services/cd-service/pkg/repository/repository.go

```diff
@@ -1098,30 +1097,6 @@ func (s *State) ProcessQueue(ctx context.Context, fs billy.Filesystem, environme
 			// whenever the next deployment happens:
 			err = s.DeleteQueuedVersion(environment, application)
 			return fmt.Sprintf("deleted queued version %d because it was already deployed. app=%q env=%q", *queuedVersion, application, environment), err
```

Comment:
Why not remove the entire `ProcessQueue` function or make it `return "", nil`?

Reply:
We still need it to delete the queue if necessary